<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lock</class>
 <widget class="QWidget" name="Lock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(186, 189, 182);</string>
  </property>
  <widget class="QPushButton" name="Com_Lock_btUnlock">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Com_Lock_lbUser">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Com_LocklePassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 245, 247);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Com_Lock_lbMessage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Com_Lock_lbBack">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Com_Lock_lbBack</zorder>
  <zorder>Com_Lock_btUnlock</zorder>
  <zorder>Com_Lock_lbUser</zorder>
  <zorder>Com_LocklePassword</zorder>
  <zorder>Com_Lock_lbMessage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
